src/components/container/ConnectDevice/ConnectDevice.tsx

```diff
@@ -112,7 +113,7 @@ export default function (props: ConnectDeviceProps) {
 	}
 
 	return (
-		<div className="mdhui-connect-device">
+		<div className="mdhui-connect-device" ref={props.innerRef}>
```

Comment:
I can't comment high enough in the diff, but line `106` probably needs the `ref` as well.